src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_latent_upscale.py

```diff
@@ -94,7 +94,7 @@ def __init__(
             scheduler=scheduler,
         )
         self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
-        self.image_processor = VaeImageProcessor(vae_scale_factor=self.vae_scale_factor)
+        self.image_processor = VaeImageProcessor(vae_scale_factor=64, resample="bicubic")
```

Comment:
Hmmm why do we do this change here?

Reply:
just to match how the  `preprocess` method was implemented for this pipeline https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_upscale.py#L46
it use default resample ("bicubic") vs the img2img preprocess use "lanczos" https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py#L85
I didn't make the change before because we were just using the VaeImageProcessor for `postprocess`

Reply:
I think this is not correct however. The `vae_scale_factor` is not 64 here but should be `self.vae_scale_factor` which corresponds to 8. The 64 was just used for images that had sizes that are **not** multiple of 64, but we can change this to 8.
```suggestion
        self.image_processor = VaeImageProcessor(vae_scale_factor=self.vae_scale_factor, resample="bicubic")
```

Reply:
Since this PR: https://github.com/huggingface/diffusers/pull/505 all stable diffusion models can work with images just being multiples of 8